Week2/alaa.js

```diff
@@ -0,0 +1,97 @@
+//1
+console.log('Hallo', 'wereld!'); // Dutch
+console.log('hola', 'mundo!'); //spanish
+console.log('مرحبا', 'العالم!'); //Arabic
+//2
+console.log("I'm awesome"); // should use two quotations instead of one
+//3
+var x;
+console.log('the value of my variable x will be: undefined');
+x = 7; // initialize the value of x
+console.log('the value of my variable x will be 7');
+console.log(x);
```

Comment:
This is correct but if you remember from our class, we should always strive to use `const` keyword first followed by `let` and then `var`. You should try to avoid `var` keyword as much as possible. In this particular case, using `const` wouldn't work because we need to re-assign `x = 7` in line 10. Thus, you can use `let x;` instead of `var`. 🙂